WatchApp Extension/Managers/ComplicationChartManager.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  ComplicationChartManager.swift
+//  WatchApp Extension
+//
+//  Created by Michael Pangburn on 10/17/18.
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import HealthKit
+import WatchKit
+
+
+private enum ComplicationChartConstants {
```

Comment:
The empty enum namespace only makes the access longer. Prefer either private extensions on the type (so the type name can be omitted when used) or just use top-level private constants instead.